tools/regions.py

```diff
@@ -168,6 +170,9 @@ def merge_region_list(
             pad_size = start - begin
             merged.puts(begin, padding * pad_size)
             begin = stop + 1
+        if restrict_size is not None and pad_to_restrict_size:
+            pad_size = int(restrict_size, 0) - begin
```

Comment:
This would restore the output to something similar to pre 11.1 behaviour

```suggestion
            pad_size = int(restrict_size, 0) - begin
            notify.info("  Padding region with 0x%x bytes" % pad_size)
```